### ISSUE TITLE
Label (open in) is not associated with same window radio button.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#129]
User experience:
If, the labels for form elements are absent, screen reader users do not know the input data expectations. Screen readers cannot programmatically determine information about input objects without an established label relationship (or redundant text serving as a label). The absence of labels prevent fields from receiving focus when read by screen readers, and users with impaired motor control do not get the benefit of a larger clickable area for the control since clicking the label activates the control.

Note: User credentials should NOT be included in the bug.

Pre-requisites:

1. Open developer toolbar via pressing F12 key.
2. Open Narrator via pressing ctrl+window+enter key.
 

Repro steps:

1. Open Azure portal with valid credentials and create a API Management.
2. Click Developer Portal button to navigate to the portal.
3. Wait portal to load in design mode
4. Navigate to the Left nav and select Pages button.
5. Navigate to the Pages window. Select "APIs: List" option.
6. Navigate to the any editable control details and select Add widget.
7. Navigate to the List of API and select it.
8. Navigate to the various controls of the "List of API" dialog box.
9. Navigate to the apis details control and press enter/spacebar.
10. Navigate to the hyperlink dialog box and reach radio button same window.
11. Verify, whether the same window radio button is associated with label (open in) or not.

Actual Result:

Label (open in) is not associated with same window radio button. Instead Screen Reader announce same window radio button 1 of 1.

Expected Result:

Label (open in) should be associated with same window radio button. So, that the Screen Reader user does able to get proper information of  same window radio button i.e open in same window radio button  1 of 2.

Note:

1. Similar issue exist for new window radio button.
2. This issue also exist on chrome browser.
3. Screen Reader does not announce the position of radio button properly i.e on same window radio button and new window radio button Screen Reader announces radio button 1 of 1.
 

MAS Reference:

MAS 3.3.2 - Labels or Instructions

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it      back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.

![image](https://user-images.githubusercontent.com/2320302/79021426-513fbe00-7b30-11ea-8aba-3bc313d0a906.png)

**Comment by azaslonov** [comment]
Fixed.
